manifests/params.pp

```diff
@@ -36,6 +36,11 @@
   $rdeck_home = '/var/lib/rundeck'
   $service_logs_dir = '/var/log/rundeck'
 
+  $rdeck_uuid = $::serialnumber ? {
```

Comment:
Oh yeah, was thinking about older Puppet compatibility but we dont need to do that now! 😄